grammar.js

```diff
@@ -8,7 +8,7 @@ module.exports = grammar({
 
   extras: $ => [
     $.comment,
-    /[\s\uFEFF\u2060\u200B\u00A0]/
+    /[\s\p{Zs}\uFEFF\u2060\u200B]/,
```

Comment:
It would be nice to include a justification for the choice of these particular characters. I was wondering about the 3 extra characters \uFEFF \u2060 \u200B because they're [not mentioned by MDN as whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace).

Reply:
I haven't gone too far down the unicode rabbit hole yet, but I'm thinking maybe let's stick with `\s` and `\p{Zs}` for the sake of simplicity 😅 (`\uFEFF` is listed in the link, but it's already covered by `\s`)

Reply:
Should we change the regex to just `/[\s\p{Zs}]/` then?

Reply:
Oh right, yes, we should.

Reply:
Oh wait, sorry. Two of the formerly-included characters: (`\u2060` *word joiner* and `\u200B` *zero-width space*) are not covered by either `\s` or `\p{Zs}`. I guess we should keep those individual characters in the regex as well.

Reply:
Done, also added back `\uFEFF` since it's actually not included in `\s` in Rust (tree-sitter/tree-sitter#67)

Reply:
Thanks, sorry for the misleading suggestion.